static/static.js

```diff
@@ -280,7 +280,24 @@ $("#greyOutUnderNav").click(function() {
 
 var lastH3 = null;
 var bounceAnimTO = null;
+var debounce = false;
+var deferred = false;
 $(window).scroll(function() {
+  // HACK: This is bad. But I can't figure out a cleaner solution without a major refactor
+  // See issue https://github.com/donejs/bit-docs-donejs-theme/issues/48
+  if(debounce){
+    deferred = true;
+    return;
+  }
+  debounce = true;
+  setTimeout(function(){
+    debounce = false;
+    if(deferred){
+      deferred = false;
+      $(window).scroll();
+    }
+  }, 500);
```

Comment:
I would either bring in `lodash` or `lodash.debounce`, or move this into a separate util file so it could be reused elsewhere